### ISSUE TITLE
Package does not work with the latest version of sequelize

### ISSUE THREAD

**Issue by rezaalavi** [lalalilo/express-sequelize-crud#47]
I get the following error trying to install the package:

npm ERR! Found: sequelize@6.6.2
npm ERR! node_modules/sequelize
npm ERR!   sequelize@"^6.5.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer sequelize@"^5.0.0" from express-sequelize-crud@6.1.5

**Comment by LucusWebsites** [comment]
Having the same problem:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: **snip**@1.0.0
npm ERR! Found: sequelize@6.6.2
npm ERR! node_modules/sequelize
npm ERR!   sequelize@"^6.3.5" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer sequelize@"^5.0.0" from express-sequelize-crud@6.1.5
npm ERR! node_modules/express-sequelize-crud
npm ERR!   express-sequelize-crud@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

**Comment by LucusWebsites** [comment]
Been using express-sequelize-crud for a couple of days now and it seems to work fine so far with sequelize 6.3.5

**Comment by urakozz** [comment]
It should be resolved when #69 will be published
